views/index.html

```diff
@@ -45,6 +45,14 @@ <h2 class="govuk-heading-m">Before you start</h2>
         <li>to upload evidence to support the objection, for example invoices showing the company is still trading or owes a debt</li>
       </ul>
 
+      {{ govukDetails({
+        summaryText: "What is The Gazette?",
```

Comment:
how about a unit test to 'GET' the page and check it contains the text - What is The Gazette - just to cover it off if it got accidentally deleted in future.